R/metadata.R

```diff
@@ -46,10 +54,17 @@ setClass(
     load.rate.units="character",
     # Other
     station="character",
+    wq.sta.id="character",
+    station.lat="ANY",
```

Comment:
the prototype does need to match the required type. could you give it another shot? NA is probably an option, or maybe as.numeric(NA)